run-tests.js

```diff
@@ -30,17 +30,13 @@ const langKeys = {
     '--php-async': false,    // run php async tests only,
 }
 
-const optionKeys = {
-    '--warnings': false,
-    '--info': false,
-}
-
```

Comment:
we actually nowhere used `optionKeys` variable down (only console.log-ed it later)